ERROR - 2024-10-03 01:33:19 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_server', 3306, '', 0)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#10 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#7 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-10-03 01:33:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: users/data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Users.php(38): App\Models\UserModel->getusers()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\server-webservice\\public\\index.php')
ERROR - 2024-10-03 01:35:47 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_server', 3306, '', 0)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#10 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#7 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-10-03 01:35:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: users/data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Users.php(38): App\Models\UserModel->getusers()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\server-webservice\\public\\index.php')
ERROR - 2024-10-03 01:38:55 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'db_server', 3306, '', 0)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#10 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\server-webservice\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\server-webservice\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\server-webservice\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\server-webservice\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\server-webservice\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\server-webservice\app\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\server-webservice\app\Controllers\Users.php(38): App\Models\UserModel->getusers()
#7 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(933): App\Controllers\Users->getUsers()
#8 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\server-webservice\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\server-webservice\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\server-webservice\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\server-webservice\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\server-webservice\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-10-03 01:38:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: users/data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\UserModel.php(17): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Users.php(38): App\Models\UserModel->getusers()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->getUsers()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\server-webservice\\public\\index.php')
CRITICAL - 2024-10-03 03:10:16 --> ParseError: syntax error, unexpected token "}"
[Method: GET, Route: users/data]
in APPPATH\Controllers\Users.php on line 105.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\server-webservice\\app\\Controllers\\Users.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Users')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Users')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Users', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\server-webservice\\public\\index.php')
CRITICAL - 2024-10-03 04:00:13 --> ErrorException: Undefined variable $usersModel
[Method: GET, Route: users/data/]
in APPPATH\Controllers\Users.php on line 70.
 1 APPPATH\Controllers\Users.php(70): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usersModel', 'C:\\xampp\\htdocs\\server-webservice\\app\\Controllers\\Users.php', 70)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->getUsers()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\server-webservice\\public\\index.php')
